tests/test_other.py

```diff
@@ -1813,6 +1813,13 @@ def test_libpng(self):
     output = self.run_js('a.out.js')
     self.assertContained('libpng passes test', output)
 
+  def test_libpng_with_pthreads(self):
```

Comment:
If you mark this tests with `@node_pthreads` then it won't need to explicit node_args